index.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = Math.random();
+  }
+}
+
+class Library {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook(book) {
+    this.data.push(book);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `concat` over `push`, which doesn't mutate the array.

Reply:
We have implemented the change :recycle: